cmd/inspect/inspect_policy.go

```diff
@@ -82,27 +81,32 @@ func inspectPolicyCmd() *cobra.Command {
 				return fmt.Errorf("invalid value for --output '%s'. accepted values: %s", outputFormat, strings.Join(validFormats, ", "))
 			}
 
+			ctx := cmd.Context()
+			afs := utils.FS(ctx)
```

Comment:
I've added this on https://github.com/hacbs-contract/ec-cli/pull/514, noting as there will be a conflict.